frontend/src/hooks/useDownload.ts

```diff
@@ -0,0 +1,126 @@
+import { RefObject, useState } from "react";
```

Comment:
This is an existing minor issue but the downloaded datafile shows text "null" instead of an empty string. Not sure how many columns this affects but I noticed it with the sample cohort ids column. Would you mind adding a fix for this here?

Reply:
This has been fixed in the [newly added commit](https://github.com/mskcc/smile-dashboard/pull/239/commits/24c56f92b93081035a1949307db5b77e79437990).

This is an existing issue not related to the frontend code refactor. Basically the issue was the following query snippet, which converts the `null` value into a "null" string:

```neo4j
apoc.text.join(sampleCohortIds, ", ") AS sampleCohortIds
```

Fixed by filtering out `null` values first:

```neo4j
coalesce(apoc.text.join([id IN sampleCohortIds WHERE id IS NOT NULL], ', '), '') AS sampleCohortIds
```